server/hedley/modules/custom/hedley_chw/hedley_chw.module

```diff
@@ -288,12 +288,12 @@ function hedley_chw_participate_in_group($child_id, $adult_id, $group_id) {
  * @throws \EntityMetadataWrapperException
  */
 function hedley_chw_get_village_residents($nid, $range = 2000) {
-  $wrapper = entity_metadata_wrapper('node', $nid);
-  $province = $wrapper->field_province->value();
-  $district = $wrapper->field_district->value();
-  $sector = $wrapper->field_sector->value();
-  $cell = $wrapper->field_cell->value();
-  $village = $wrapper->field_village->value();
+  $node = node_load($nid);
```

Comment:
what was the issue with EntityMetadataWrapper?

Reply:
@AronNovak  Memory leakage.
When executed from Drush script, every time this function was invoked, we'd get +2M of RAM.
After ~120 iterations, we run out of memory. 
Tried `drupal_static_reset()` and some other stuff, like `unset()`, but it made no difference.
Maybe you got other idea?